lib/IR/GraphScheduler.cpp

```diff
@@ -48,14 +48,18 @@ Scheduler *createScheduler(SchedulerKind schedulerKind, Function &G,
 
 void IRFunction::scheduleGraph(NodesPtrList &Schedule) {
   Schedule.clear();
-  for (auto &N : G_->getParent()->getConstants()) {
+  for (auto &N : G_->getConstants()) {
```

Comment:
Since there can be duplicates in the returned list, aren't you adding duplicate Constants here if multiple Nodes refer to them? Maybe `getConstants()` should return an `unordered_set`?

Reply:
Since moving to walking the modules placeholders and constants I don't think we can add a duplicate anymore. So we should be good.